source/jormungandr/jormungandr/otlp.py

```diff
@@ -218,9 +218,17 @@ def send_event_metrics(self, event_type: str, params: Dict = {}) -> None:
         if not self._meter:
             return
 
-        labels = {"platform": self.__platform, "account": self.__account, "event_type": event_type}
-        labels.update(params)
+        keep_labels = ["call", "event_type", "mode", "coverage", "service", "status"]
+        labels = {
+            "platform": self.__platform,
+            "account": self.__account,
+            "event_type": event_type,
+            "instance_id": self.__instance_id,
+        }
 
+        for keep_label in keep_labels:
+            if keep_label in params:
+                labels[keep_label] = params.pop(keep_label)
```

Comment:
```suggestion
        for keep_label in set(keep_labels) & set(params):
                labels[keep_label] = params.pop(keep_label)
```
mais on s'en fout...

Reply:
see together, I will keep the current version, because I tested it on sandbox